src/main/java/com/faforever/client/chat/ChatChannelUser.java

```diff
@@ -55,6 +55,7 @@ public class ChatChannelUser {
   private final ObjectProperty<Image> gameStatusImage = new SimpleObjectProperty<>();
   private final StringProperty statusTooltipText = new SimpleStringProperty();
   private final BooleanProperty displayed = new SimpleBooleanProperty(false);
+  private final StringProperty playerNote = new SimpleStringProperty();
```

Comment:
Why add a separate player note attribute to the chatChannelUser instead of using the one already in the associated player?

Seems like you could avoid a binding by just returning the actual playernote property in the getter and setter.

Reply:
> in the getter.

Do you mean in `ChatChannelUser` class?

If there is no player? What should I return?

Reply:
If there is no player we should just return an empty property or null.

We can't truly track irc users so it doesn't make sense to set notes on them since they will likely get lost